quint/src/quintVerifier.ts

```diff
@@ -142,30 +142,33 @@ function err<A>(explanation: string, errors: ErrorMessage[] = [], traces?: ItfTr
 }
 
 function findApalacheDistribution(): VerifyResult<ApalacheDist> {
-  const configuredDist =
-    process.env.APALACHE_DIST && path.isAbsolute(process.env.APALACHE_DIST)
-      ? process.env.APALACHE_DIST
-      : path.join(process.cwd(), process.env.APALACHE_DIST!)
-
-  // TODO: fetch release if APALACHE_DIST is not configured
-  // See https://github.com/informalsystems/quint/issues/701
```

Comment:
Can we keep this TODO, it is still relevant.

Reply:
Ah, sorry, overlooked this in the diff! Restored in 4234a021